src/labels/CustomLabels.labels

```diff
@@ -6065,6 +6065,30 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Failure message if an invalid base64 url was provided</shortDescription>
         <value>The base64url format is invalid. It can only contain these characters: [a-z A-Z 0-9 _ -]</value>
     </labels>
+    <labels>
+        <fullName>gePaymentProcessErrorGeneric</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>General failure message description for tokenization or purchase errors</shortDescription>
+        <value>There was an error processing the transaction. Please review the error and try again. Alternatively, select Do Not Charge Card to enter a pledged donation without processing a card payment.</value>
+    </labels>
+    <labels>
+        <fullName>gePaymentProcessingErrorBanner</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Banner error message for card processing error.</shortDescription>
+        <value>There was an error processing the transaction.</value>
+    </labels>
+    <labels>
+        <fullName>gePaymentServicesErrorResponse</fullName>
```

Comment:
@dfuller114 and @ptewson-sfdo This custom label should be deleted.

Reply:
deleted